pkg/cloud/msd/workmftmsd.go

```diff
@@ -0,0 +1,107 @@
+package work_mft_msd
```

Comment:
This seems a bit cryptic plus I don't think the linters like `_` in a package name.  How about we name this 

```suggestion
package machineset
```

and ....